### DISCUSSION TITLE
Option to hide app icon from launcher

### DISCUSSION THREAD

**Discussion by Xisrr1** [breezy-weather/breezy-weather#1041] [Ideas]
This would be a nice addition since I use the widget, like Samsung and Google weather. @papjul

**Comment by papjul** [comment]
What do you mean "hide"? Just remove the icon from your homescreen?

**Reply by Xisrr1** [reply]
> What do you mean "hide"? Just remove the icon from your homescreen?

From the launcher.

**Reply by papjul** [reply]
Hold your finger on the app icon and drag it to remove it. Some launchers may work.differently, please refer to your launcher documentation in that case

**Reply by Xisrr1** [reply]
OneUI launcher. I mean to fully hide it even from the app drawer. ![Screenshot_20240506_124801_Weather.jpg](https://github.com/breezy-weather/breezy-weather/assets/133952156/b15199de-da19-4115-b519-93b10048e53f)

**Reply by papjul** [reply]
We have no reason to hide the app icon from the drawer where all available apps are. If you really need it, it must be implemented on launcher app side, please contact them or use a launcher that already supports it

**Reply by papjul** [reply]
Note that what you request is no longer supported since Android Q: https://developer.android.com/reference/android/content/pm/LauncherApps#getActivityList(java.lang.String,%20android.os.UserHandle)
